api/src/main/resources/Reference_Application_Concepts-23.xml

```diff
@@ -835,11 +843,10 @@
   <concept_name concept_id="163139" name="Quartenary" locale="en" creator="1" date_created="2015-12-13 15:00:18.0" concept_name_id="134811" voided="false" voided_by="1" uuid="134811BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
   <concept_name concept_id="163139" name="Quaternaire" locale="fr" creator="1" date_created="2015-12-13 15:02:03.0" concept_name_id="134812" voided="false" voided_by="1" uuid="134812BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
   <concept_name concept_id="163139" name="Kwat&#232;rn&#232;r" locale="ht" creator="1" date_created="2015-12-13 15:02:03.0" concept_name_id="134813" voided="false" voided_by="1" uuid="134813BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
-  <concept_name concept_id="165095" name="General patient note" locale="en" creator="1" date_created="2018-04-18 14:34:20.0" concept_name_id="141526" voided="false" voided_by="1" uuid="141526BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
-  <concept_map_type concept_map_type_id="1" name="SAME-AS" creator="1" date_created="2018-05-03 00:00:00.0" changed_by="1" is_hidden="false" retired="false" retired_by="1" uuid="35543629-7d8c-11e1-909d-c80aa9edcf4e"/>
-  <concept_map_type concept_map_type_id="2" name="NARROWER-THAN" creator="1" date_created="2018-05-03 00:00:00.0" changed_by="1" is_hidden="false" retired="false" retired_by="1" uuid="43ac5109-7d8c-11e1-909d-c80aa9edcf4e"/>
-  <concept_map_type concept_map_type_id="3" name="BROADER-THAN" creator="1" date_created="2018-05-03 00:00:00.0" changed_by="1" is_hidden="false" retired="false" retired_by="1" uuid="4b9d9421-7d8c-11e1-909d-c80aa9edcf4e"/>
```

Comment:
Is this just a formatting change? I do not see any difference between the red and green lines.

Reply:
Can we fix the dates to be the same?

Reply:
@dkayiwa Reverted those lines back to the previous one